packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -186,6 +189,16 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     return (await this.mediatorSparqlSerialize.mediate({ context, handle, handleMediaType: mediaType })).handle;
   }
 
+  /**
+   * Invalidate all internal caches related to the given page URL.
+   * If no page URL is given, then all pages will be invalidated.
+   * @param {string} pageUrl The page URL to invalidate.
+   * @return {Promise<any>} A promise resolving when the caches have been invalidated.
+   */
+  public invalidateHttpCache(pageUrl?: string): Promise<any> {
```

Comment:
Can this function be called on the newEngine object, or where is it used?

Reply:
Yep! Because the `newEngine()` returns `ActorInitSparql`.